### DISCUSSION TITLE
[@types/gapi] Old method deprecaited, needs to add the new method: gapi.client.load(urlOrObject)

### DISCUSSION THREAD

**Discussion by BenJackGill** [DefinitelyTyped/DefinitelyTyped#60403] [Issues with a @types package]
The [reference docs for gapi](https://github.com/google/google-api-javascript-client/blob/master/docs/reference.md) show that one of the methods has been deprecaited and replaced. But this has not been updated for `@types/gapi`.

The **[deprecaited method](https://github.com/google/google-api-javascript-client/blob/master/docs/reference.md#----gapiclientloadname----version----callback--)** is `gapi.client.load(name, version, callback)`. If a callback is not provided, a [goog.Thenable](https://google.github.io/closure-library/api/goog.Thenable.html) is returned.

The **[new method](https://github.com/google/google-api-javascript-client/blob/master/docs/reference.md#----gapiclientloadurlorobject--)** is `gapi.client.load(urlOrObject)`. It always returns a Promise-like [goog.Thenable](https://google.github.io/closure-library/api/goog.Thenable.html) object.

I'm a novice coder still learning TypeScript, so I don't know how make "pull requests" to have this change added.

Can someone else with more knowledge take this over?

[a comment by an automated account is not shown]

**Comment by sgtfrankieboy** [comment]
I recommend that you try out the `@types/gapi.client` package.

The `@types/gapi` and `@types/gapi.youtube` etc should be deprecated in favor of the automatically generated typing files under the `@types/gapi.client.*` naming.

**Reply by BenJackGill** [reply]
Thanks! Maybe would be good to add this to the NPM page?

**Reply by BenJackGill** [reply]
Actually, after trying `@types/gapi.client` I can also see that it has the same problem. It uses types for the depreciated method, not the new one.
